### ISSUE TITLE
Video OSD loading. 

### ISSUE THREAD

**Issue by JFG1000** [jurialmunkey/skin.arctic.fuse#992]
### Skin section

Video OSD

### Current Behavior

Hi jurial for a while now I've noticed it's taking a short while to load the options to enable/disable things like subtitles etc when hovering over the icons on the video OSD, it used to be pretty instant, not sure if you are aware of this. 
![20240704_204651](https://github.com/jurialmunkey/skin.arctic.fuse/assets/76907836/1e8a58ac-aa31-4a6c-8144-de1d8703c1bd)

### Expected Behavior

Usually when hovering over icons, subtitles etc it loads the selection box instantly. 

### Steps To Reproduce

Bring up video osd 

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.

**Comment by jurialmunkey** [comment]
Not much to do with the skin.

These are plugin directories from skin variables (just like any other widget). Skin variables asks kodi via json rpc for details about the current stream.

So it could be delayed by either normal things which delay widgets loading (eg lots of other widgets loading blocking up the queue) and/or a slow response from kodi in querying the video player for stream details (all sorts of things could cause this).

Basically there's nothing the skin can do - it's just saying to kodi "load this plugin content here please" and then that takes however long until kodi fulfils that request.
